src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,72 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+/* eslint-disable import/no-extraneous-dependencies */
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
+
+export const fetchRockets = createAsyncThunk('books/fetchBooks', async () => {
+  try {
+    const response = await axios.get(`${url}`);
+    return response.data;
+  } catch (err) {
+    return err.message;
+  }
+});
+
+const initialState = {
+  rocketList: [
+  ],
+  status: 'idle',
+};
+
+/* eslint no-param-reassign: "error" */
+
+const rocketSlice = createSlice({
```

Comment:
- [ ] Let's try to not disable Linter checks in our program as they are there to help us write better and more professional code. Disabling Linter checks defies its purpose and doesn't help you become a professional developer. Learning how to fix them now will set you up ready for the industry as that's the standard. Please fix it 👍🏽